core-services/sanitize-prow-jobs/_config.yaml

```diff
@@ -1953,3 +1953,36 @@ groups:
     - pull-ci-openshift-cluster-api-provider-ovirt-release-4.5-e2e-ovirt
     - pull-ci-openshift-cluster-api-provider-ovirt-release-4.6-e2e-ovirt
     - pull-ci-openshift-cluster-api-provider-ovirt-release-4.7-e2e-ovirt
+  vsphere:
+    jobs:
```

Comment:
maybe we use something like https://github.com/openshift/release/blob/7e0d86a001a1bc06fd0546ac38a2207fb70f4d2e/core-services/sanitize-prow-jobs/_config.yaml#L1930-L1933

where we mark all `pull-ci-*e2e-vpshere*` to this build cluster??

Reply:
Yeah, the paths ie file names wouldn't make sense.. but we need something something more easy to manage. 

> If we want to improve how this is configured after I am all for it.

Who will be responsible for making that happen? As long as we have an owner I am not against moving ahead for now.

Reply:
So far its been myself but I don't think that is scaleable for one person to be responsible for it.
@stevekuznetsov or @hongkailiu is there a better way to configure which cluster is sent a specific job?